photon-core/src/main/java/org/photonvision/common/configuration/ConfigManager.java

```diff
@@ -28,7 +28,8 @@
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
 import java.time.temporal.TemporalAccessor;
-import java.util.*;
+import java.util.Date;
+import java.util.List;
```

Comment:
Can we have lint do this automatically in the future? Like can spotless forbid star imports?

Reply:
It's not necessarily removing star imports, just doesn't use them when it really isn't needed, like 2 items. You can see there are cases where I did use it when there were 4-5